### PR TITLE
Release highlights for the week ending 2021-12-03

### PR DESCRIPTION
**Pull request by PrashantR31**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/changes/highlights/2021-12-03.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+date: 2021-12-03T10:00:00
+title: Highlights for the week ending December 03, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
```
Comment (ubergesundheit) [inline src/content/changes/highlights/2021-12-03.md:11]:
```suggestion

- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#250---2021-11-29) v2.5.0 updates the controller image to upstream [v1.1.0](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.1.0).
```

File: src/content/changes/highlights/2021-12-03.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## User interfaces
+ - Version caching is now disabled for the selfupdate command, so that the latest version can be seen right after it’s released.
+ - Make the --release flag mandatory, which specifies the workload cluster release version in the template cluster and template nodepool subcommands.
+
+## Documentation
+ - Docs for --release flag added
```
Comment (snizhana-dynnyk) [inline src/content/changes/highlights/2021-12-03.md:18]:
```suggestion

```

Comment (snizhana-dynnyk) [reply src/content/changes/highlights/2021-12-03.md:18]: I wouldn’t mention this @PrashantR31 
We usually include changes in docs if the article had significant updates or a new doc article was added.

Comment (PrashantR31) [reply src/content/changes/highlights/2021-12-03.md]: Sure, I've removed it.

File: src/content/changes/highlights/2021-12-03.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+## Apps
+
+- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) [v2.5.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#250---2021-11-29) updates the controller image to upstream [v1.1.0](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.1.0). [v2.6.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#260---2021-12-02) adds the ability to specify a custom LoadBalancer Service name suffix.
+
+
+## User interfaces
+ - Version caching is now disabled for the selfupdate command, so that the latest version can be seen right after it’s released.
+ - Make the --release flag mandatory, which specifies the workload cluster release version in the template cluster and template nodepool subcommands.
+
+## Documentation
```
Comment (snizhana-dynnyk) [inline src/content/changes/highlights/2021-12-03.md:19]:
```suggestion

```

File: src/content/changes/highlights/2021-12-03.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+## Apps
+
+- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) [v2.5.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#250---2021-11-29) updates the controller image to upstream [v1.1.0](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.1.0). [v2.6.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#260---2021-12-02) adds the ability to specify a custom LoadBalancer Service name suffix.
+
+
+## User interfaces
+ - Version caching is now disabled for the selfupdate command, so that the latest version can be seen right after it’s released.
+ - Make the --release flag mandatory, which specifies the workload cluster release version in the template cluster and template nodepool subcommands.
```
Comment (snizhana-dynnyk) [inline src/content/changes/highlights/2021-12-03.md:17]: not sure if this is important change to include, but if you think so, please also specify that this is a flag for `kubectl gs` :)

**Review by snizhana-dynnyk — CHANGES_REQUESTED** [review]

File: src/content/changes/highlights/2021-12-03.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+- [kong-app](https://github.com/giantswarm/kong-app/) [v2.3.4](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#234---2021-12-02) adds further Giant Swarm monitoring.
+- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) [v2.5.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#250---2021-11-29) updates the controller image to upstream [v1.1.0](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.1.0).
+- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) [v2.6.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#260---2021-12-02) adds the ability to specify a custom LoadBalancer Service name suffix.
+
+
+## User interfaces
+ - Version caching is now disabled for the selfupdate command, so that the latest version can be seen right after it’s released.
+ - Make the --release flag mandatory, which specifies the workload cluster release version in the template cluster and template nodepool subcommands.
```
Comment (marians) [inline src/content/changes/highlights/2021-12-03.md:20]: I do think this one is important, as the command will fail without the `--release` flag.

```suggestion
 - In kubectl-gs we had to make the `--release` flag mandatory, which specifies the workload cluster release version in the `template cluster` and `template nodepool` subcommands.
```

File: src/content/changes/highlights/2021-12-03.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+- [aws-load-balancer-controller-app](https://github.com/giantswarm/aws-load-balancer-controller-app) [v1.0.1](https://github.com/giantswarm/aws-load-balancer-controller-app/blob/main/CHANGELOG.md#101---2021-12-02) adds further Giant Swarm monitoring.
+- [kong-app](https://github.com/giantswarm/kong-app/) [v2.3.4](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#234---2021-12-02) adds further Giant Swarm monitoring.
+- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) [v2.5.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#250---2021-11-29) updates the controller image to upstream [v1.1.0](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.1.0).
+- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) [v2.6.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#260---2021-12-02) adds the ability to specify a custom LoadBalancer Service name suffix.
+
+
+## User interfaces
+ - Version caching is now disabled for the selfupdate command, so that the latest version can be seen right after it’s released.
```
Comment (snizhana-dynnyk) [inline src/content/changes/highlights/2021-12-03.md:19]:
```suggestion

```

File: src/content/changes/highlights/2021-12-03.md (hunk 3 above)
Comment (PrashantR31) [reply src/content/changes/highlights/2021-12-03.md]: Oh, this was removed as well.

File: src/content/changes/highlights/2021-12-03.md (hunk 4 above)
Comment (PrashantR31) [reply src/content/changes/highlights/2021-12-03.md]: kubectl gs added

File: src/content/changes/highlights/2021-12-03.md (hunk 5 above)
Comment (PrashantR31) [reply src/content/changes/highlights/2021-12-03.md]: Alright, let me remove it.

**Review by marians — CHANGES_REQUESTED** [review]
Looks like the actual content has disappeared

**Review by snizhana-dynnyk — APPROVED** [review]

**Comment by snizhana-dynnyk** [comment]
restored the content from the slack announcement, @marians could you please approve so we can merge this :)

**Review by marians — APPROVED** [review]

**Pull request merged**